website/src/pages/docs/features/subscriptions.mdx

```diff
@@ -161,7 +161,7 @@ class SSELink extends ApolloLink {
     url.searchParams.append('query', print(operation.query))
     if (operation.operationName) {
       url.searchParams.append(
-        'operationName',
```

Comment:
I don't understand what is the motivation behind this.

Reply:
Creating subscriptions with the Apollo Client seem to fail with the default example code. A 400 is returned with the following:

```
{"data":null,"errors":[{"message":"Could not determine what operation to execute.","extensions":{}}]}
```

Changing `operationName` to `operation` fixed the issue.

Reply:
It is `operationName `in GraphQL over HTTP spec. Could you share a reproduction on CodeSandbox or StackBlitz?